src/WinRT.Runtime/ComWrappersSupport.cs

```diff
@@ -134,6 +137,11 @@ public static ObjectReference<T> GetObjectReferenceForInterface<T>(IntPtr extern
             return GetObjectReferenceForInterface<T>(externalComObject, iid, true);
         }
 
+        public static IObjectReference GetObjectReferenceForInterface(IntPtr externalComObject, Guid iid, bool requireQI)

```

Comment:
Nit: it'd be nice to add proper XML docs to these APIs, and in particular for this one we might want to call out in the remarks that callers have the responsibility of passing valid `iid` values for the input objects, otherwise the behavior will be weird. That is, because if you pass `requireQI: false`, the call will still succeed, but then you would get a failure potentially much later on when the implementation is trying to use that `iid` to get an agile reference from somewhere else entirely.

Reply:
Added some docs here.